Firestore/core/src/remote/bloom_filter.h

```diff
@@ -63,6 +82,14 @@ class BloomFilter final {
 
   // Bloom filter's bitmap.
   std::vector<uint8_t> bitmap_;
+
+  BloomFilter::Hash Md5HashDigest(const absl::string_view key) const;
+
+  int32_t GetBitIndex(const BloomFilter::Hash& hash,
+                      int32_t i,
+                      int32_t bit_count) const;
+
+  bool IsBitSet(const std::vector<uint8_t>& bitmap, int32_t index) const;
```

Comment:
Does `IsBitSet()` need to take the `std::vector<uint8_t>` as an argument? Can it just access the `bitmap_` member variable?